### PR TITLE
Added Offset Query Spec

### PR DESCRIPTION
**Pull request by drewsonian**
I couldn't find a place where you implemented setFirstResult() and perhaps there's a better way to do paging, but I thought this would be helpful.  Let me know if there's a different way to do paging or if this should exist in a different folder.  I'm pretty new to Github / pull requests / etc, so if I'm doing anything wrong, let me know.

Thanks!

### CODE REVIEW

**Comment by Nyholm** [comment]
Thank you for your PR. This is missing and needed. 

You have missed some code style things. I'll make some inline comments. 

PS. Have a look at this tool to automatically fix coding standards: https://github.com/FriendsOfPHP/PHP-CS-Fixer

File: src/Query/Offset.php [hunk 1]
```diff
@@ -0,0 +1,25 @@
+<?php
+namespace Happyr\DoctrineSpecification\Query;
```
Comment (Nyholm) [inline src/Query/Offset.php]: Make sure there is an empty line before and after "namespace" and "use"

File: src/Query/Offset.php [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+use Doctrine\ORM\QueryBuilder;
+class Offset implements QueryModifier
+{
+    /**
+     * @var int offset
+     */
+    protected $offset;
+    /**
```
Comment (Nyholm) [inline src/Query/Offset.php]: Empty line needed here

File: src/Query/Offset.php [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+     * @param int $offset
+     */
+    public function __construct($offset)
+    {
+        $this->offset = $offset;
+    }
+    /**
```
Comment (Nyholm) [inline src/Query/Offset.php]: Empty line needed here

**Comment by drewsonian** [comment]
Thanks, I copied and pasted from my editor into the online editor and that messed things up somehow.  I'm going to look into that PHP-CS-Fixer, I hadn't noticed it on FOS before.

**Comment by Nyholm** [comment]
No worries. Thank you for this.

**Pull request merged**